### PR TITLE
Add documentation and scripts for 8 kHz model

### PR DESCRIPTION
**Pull request by jbalam-nv**
#Added
- Documentation for 8 kHz model in ASR docs
- Config file for a 8 khZ and a script for converting audio to 8 kHz pcm u-law format

#Removed
- Some empty References sections in docs

### CODE REVIEW

File: docs/sources/source/asr/8kHz_models.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Switchboard test       10.74
+Switchboard dev        10.59
+====================== =====================
+
+The model was first pretrained with 8 kHz LibriSpeech data for 134 epochs and then trained for another 250 epochs using G.711 audio from all the five datasets listed above. For best accuracy
+in your application, you may choose to :ref:`fine-tune <fine-tune>` this model using data collected from your application.
+
+..
+The pre-trained model is available for download `here <https://ngc.nvidia.com/models/nvidian:nemo:quartznet_15x5_8_khz_for_nemo>`_.
+
```
Comment (jbalam-nv) [inline docs/sources/source/asr/8kHz_models.rst:33]: Commented this as we are going to wait for QA to approve the model for release.

File: scripts/convert_wav_to_g711wav.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+args = parser.parse_args()
+
+def __convert_audio(in_path, out_path):
+    """
+    Helper function that's called per thread, converts wav to G.711 wav.
+    Args:
+        in_path: source wav file to convert
+        out_path: destination for G.711 wav file
+    """
+    cmd = ["sox", in_path, "-r", "8000", "-c", "1", "-e", "u-law",out_path]
```
Comment (jbalam-nv) [inline scripts/convert_wav_to_g711wav.py:35]: This script is based on existing script fisher_audio_to_wav.py. This line is the biggest difference other than comments.

File: scripts/convert_wav_to_g711wav.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2020 NVIDIA Corporation
```
Comment (blisc) [inline scripts/convert_wav_to_g711wav.py:1]: Can you add the entire copyright?

File: docs/sources/source/asr/8kHz_models.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
+8kHz Models
```
Comment (blisc) [inline docs/sources/source/asr/8kHz_models.rst:1]: Can you make a reference to your conversion script somewhere in here?

Comment (blisc) [inline docs/sources/source/asr/8kHz_models.rst:1]: Can you add a line to our changelog?

File: scripts/convert_wav_to_g711wav.py (hunk 3 above)
Comment (jbalam-nv) [reply scripts/convert_wav_to_g711wav.py]: Done! Looks like I copy pasted from an older file.

File: docs/sources/source/asr/8kHz_models.rst (hunk 4 above)
Comment (jbalam-nv) [reply docs/sources/source/asr/8kHz_models.rst:1]: Done

Comment (jbalam-nv) [reply docs/sources/source/asr/8kHz_models.rst:1]: Done, on line 11.

File: docs/sources/source/asr/8kHz_models.rst (hunk 1 above)
Comment (7SGLLC) [reply docs/sources/source/asr/8kHz_models.rst:33]: Any update on this? Would like to test the model.

**Pull request merged**